src/PowerShellEditorServices.Hosting/GlobalSuppressions.cs

```diff
@@ -0,0 +1,11 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Performance", "CA1819:Properties should not return arrays", Justification = "Cmdlet parameters can be arrays", Scope = "module")]
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Globalization", "CA1303:Do not pass literals as localized parameters", Justification = "PSES is not localized", Scope = "module")]
+
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Design", "CA1031:Do not catch general exception types", Justification = "PSCmdlet.ThrowTerminatingError() is used instead", Scope = "member", Target = "~M:Microsoft.PowerShell.EditorServices.Commands.StartEditorServicesCommand.EndProcessing")]
```

Comment:
Could these be placed inline? The ones that target a specific member that is.

Reply:
They can be. I've never liked those FxCop attributes that occur all over the code, but will leave it up to others

Reply:
I can certainly see the pros of having the attribute inline, so you get the justification like a comment

Reply:
I personally agree with @SeeminglyScience about putting them inline so they're "self documenting"

Reply:
Yeah it's far from perfect.  I hate seeing them too, but it makes it confusing why one thing warns and another doesn't.  Most folks aren't going to know to look in that file.

I usually go with `#pragma warning disable CA1031` and `#pragma warning restore CA1031` since it's slightly less aggressive, but then you lose *a lot* of information.  Probably not suitable.

Reply:
Went for the attribute where possible, just since it's got a field for `Justification`. Some things required a pragma though